staging/cse/windows/securetlsbootstrapfunc.ps1

```diff
@@ -0,0 +1,75 @@
+# Thin wrapper around [IO.Directory]::GetFiles so we can mock it as needed within Pester tests
+function GetCachedSecureTLSBootstrapClientPath {
+    Param(
+        [Parameter(Mandatory=$true)][string]
+        $SecureTLSBootstrapClientCacheDir
+    )
+
+    $result = [IO.Directory]::GetFiles($SecureTLSBootstrapClientCacheDir, "windows-amd64.zip", [IO.SearchOption]::AllDirectories)
+    return (, $result) # so that Powershell doesn't "unroll" the array returned from [IO.Directory]::GetFiles
+}
+
+# Installs the secure TLS bootstrap client from the VHD cache. If a custom URL is specified, then any cached
+# versions will be ignored and removed in favor of the custom version
+function Install-SecureTLSBootstrapClient {
```

Comment:
Would be good to have a comment on this function warning the caller of the function that the function might not exist. Kinda the opposite of "deprecated". Am thinking about the next developer who wants to use it.

Reply:
added a note